cmd/payload-testing-ui/server.go

```diff
@@ -31,24 +48,43 @@ const (
 	runsListTemplate = `
 <h1>Pull Request Payload Qualification Runs</h1>
 {{ len .Items }} run(s)
-<ul>
-{{ range .Items }}
-  <li>
-    {{ with .ObjectMeta }}
-      {{ with $url := printf "%s/%s" .Namespace .Name }}
-        <a href="` + runsURL + `{{ $url }}">{{ $url }}</a>
-      {{ end }}
-    {{ end }}
-  </li>
-{{ end }}
-</ul>
+<table class="table">
+	<thead>
+		<tr>
+			<th title="The name of the Pull Request Payload Qualification Run" class="info">Name</th>
+			<th title="The author of Pull Request and the name of Pull Request" class="info">Pull Request</th>
+		</tr>
+	</thead>
+	<tbody>
+		{{ range .Items }}
+		<tr>
+			<td>
+			{{ with .ObjectMeta }}
+				{{ with $url := printf "%s/%s" .Namespace .Name }}
+				<a href="` + runsURL + `{{ $url }}">{{ $url }}</a>
+				{{ end }}
+			{{ end }}
+			</td>
+			<td>
+			{{ with .Spec.PullRequest }}
+			<ul>
+				<li>Title: {{ prLink . }}</li>
+				<li>Author: {{ authorLink .PullRequest.Author }}</li>
+				<li>Organization: {{ orgLink .Org }}</li>
+				<li>Repository: {{ repoLink .Org .Repo }}</li>
+				<li>Number: {{ .PullRequest.Number }}</li>
```

Comment:
Let's try something more compact, along the lines of https://prow.ci.openshift.org:

```html
<thead>
    <tr>
        <th>Name</th>
        <th>Repository</th>
        <th>Pull Request</th>
    </tr>
</thead>
<tbody>
    <tr>
        <td>
            <a href="/runs/ci/00460cd0-b598-11ec-9a85-bbadead2be9e-0">ci/00460cd0-b598-11ec-9a85-bbadead2be9e-0</a>
        </td>
        <td>
            <a href="https://github.com/openshift/origin">openshift/origin</a>
        </td>
        <td>
            <a href="http://github.com/openshift/origin/pull/26973">#26973 WIP: Reenable the implementation for basic egressfw e2e CI test</a>
            by <a href="https://github.com/msherif1234">msherif1234</a>
        </td>
    </tr>
```

Reply:
We can similarly reduce the section in the details page.